lightning/api_grpc.go

```diff
@@ -35,13 +34,15 @@ func NewLndGrpcLightningAPI(getData GetDataCall) LightingAPICalls {
 		return nil
 	}
 
-	return &LndGrpcLightningAPI{
+	ret := &LndGrpcLightningAPI{
 		Client:       client,
 		RouterClient: routerClient,
 		CleanupFunc:  cleanup,
 		API:          API{GetNodeInfoFullThreshUseDescribeGraph: 500},
-		Name:         "lndgrpc",
 	}
+
+	ret.Name = "lndgrpc"
```

Comment:
why is this change needed ?

Reply:
Property `Name` is used by all implementations and was moved to "base class". And initializing it "inline" looks even more awkward.